.github/workflows/publish-technical-documentation.yml

```diff
@@ -0,0 +1,84 @@
+name: "publish-technical-documentation"
+
+on:
+  push:
+    branches:
+    - "main"
+    - "release-*"
+    tags:
+    - "mimir-[0-9]+.[0-9]+.[0-9]+"
+    paths:
+    - "docs/sources/**"
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+    - name: "Check out code"
+      uses: "actions/checkout@v3"
+    - name: "Build website"
+      run: |
+        docker run -v "${PWD}/docs/sources:/hugo/content/docs/mimir/next" --rm grafana/docs-base:latest /bin/bash -c 'make hugo'
+
+  sync:
+    runs-on: "ubuntu-latest"
+    needs: "test"
+    steps:
+
+    - name: "Checkout Mimir repo"
+      uses: "actions/checkout@v3"
+      with:
+        fetch-depth: 0
+
+    - name: "Checkout Actions library"
+      uses: "actions/checkout@v3"
+      with:
+        repository: "grafana/grafana-github-actions"
+        path: "./actions"
+
+    - name: "Install Actions from library"
+      run: "npm install --production --prefix ./actions"
+
+    - name: "Determine technical documentation version"
+      uses: "./actions/docs-target"
+      id: "target"
+      with:
+        ref_name: "${{ github.ref_name }}"
+
+    - name: "Determine latest release tag"
+      id: "latest-released"
+      shell: "bash"
+      run: |
+        tag="$(./tools/latest-tag '^mimir-(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)$')"
```

Comment:
Where is the source of `latest-tag` script/tool? Who installs it?

Reply:
I've replaced the script with its one line contents in 897b0d73c